components/date-picker/date-range-popup.component.ts

```diff
@@ -179,6 +179,12 @@ export class DateRangePopupComponent implements OnInit, OnChanges, OnDestroy {
     return this.showToday || this.hasTimePicker || !!this.extraFooter || !!this.ranges;
   }
 
+  get arrowPosition(): { left?: string; right?: string } {
+    return this.dir === 'rtl'
+      ? { right: `${this.datePickerService?.arrowLeft}px` }
```

Comment:
Try to change the style here? We can only change the `patch.less` , because the other CSS files arc synced from antd.

Reply:
thanks for review, i need get the less value of `@input-padding-horizontal-base`, i don't know how to get it in `ts` file, what about set it in `patch.less`?